proposal (copy).md

```diff
@@ -0,0 +1,139 @@
+---
+title: Pandas
+bibliography: biblio.bib
+link-citations: true
+---
+
+## Goals
+
+Fund the continued maintenance and improvements for Pandas
+(@mckinney-proc-scipy-2010), a Python library providing high-performance,
+easy-to-use data structures. Pandas is a foundational library in the Scientific
+Python Ecosystem, providing the most widely-used data structure for
+heterogenous, tabular data ().
```

Comment:
I suppose the brackets are for adding some references on this statement?

Not directly an idea for authoritative sources, but maybe the StackOverflow post on pandas a a driver for python's popularity could be used?

Reply:
Fixed in #4